python/dask_cudf/dask_cudf/tests/test_sort.py

```diff
@@ -72,7 +72,7 @@ def test_sort_repartition():
     dd.assert_eq(len(new_ddf), len(ddf))
 
 
-@xfail_dask_expr("dask-expr code path fails with nulls")
+@xfail_dask_expr("missing null support", lt_version="2024.5.1")
```

Comment:
I'd like this test to run for anything newer than the `"2024.5.0"` release (including `main`). Not sure of the "correct" way to do this.

Reply:
I think that is correct. `pytest.mark.xfail(some_condition, reason="some_reason")` which is what this expands to is the canonical way.